app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -334,6 +344,24 @@ def parse_persistent_volume(persistent_volume)
       new_result
     end
 
+    def parse_persistent_volume_claim(claim)
+      new_result = parse_base_item(claim)
+      new_result.merge!(
+        :desired_access_modes => claim.spec.accessModes,
+        :phase                => claim.status.phase,
+        :actual_access_modes  => claim.status.accessModes,
+        :capacity             => claim.status.capacity.to_h,
```

Comment:
You will need to rebase this pr based on : 
https://github.com/ManageIQ/manageiq/pull/7955/files#diff-0324981fdb3019ce6d98f9c86d97f2bbR327